Galen & Wendy's html5 as a platform assignment

gkbroderick opened the pull request.

Commit:
Wendy 1st commit.

Commit:
Added Audio files

Commit:
Cleaned up Audio Files

gkbroderick committed:
Galen first commit

gkbroderick committed:
Delete .DS_Store

gkbroderick committed:
Delete .DS_Store

gkbroderick committed:
Delete .DS_Store

Commit:
Add system file gitignore

gkbroderick committed:
Delete .DS_Store

Commit:
Add gitignore

gkbroderick committed:
HELP

gkbroderick committed:
HELP

Commit:
Merge branch 'master' of https://github.com/gkbroderick/html5-as-a-platform

Conflicts:
	.DS_Store
	Audio/.DS_Store

Commit:
delete .DS.Store from Audio and the HTML5 folder
Merge branch 'master' of https://github.com/gkbroderick/html5-as-a-platform

Commit:
delete Video/.DS_Store

 Please enter the commit message for your changes. Lines starting

gkbroderick committed:
Delete DS Store again

gkbroderick committed:
Merge branch 'master' of https://github.com/gkbroderick/html5-as-a-platform
Trying to solve DS STORE HELL

gkbroderick committed:
update main page links

gkbroderick committed:
fix Camera Code Syntax, Add READMEs

gkbroderick committed:
add READMEs, for real this time

Commit:
readme file updated

Commit:
Merge branch 'master' of https://github.com/gkbroderick/html5-as-a-platform

gkbroderick committed:
Update README

Commit:
Add change to bear

Commit:
Merge branch 'master' of https://github.com/gkbroderick/html5-as-a-platform

Commit:
bear

gkbroderick committed:
Update README.md

Commit:
Added pause-play button

Commit:
...Merge branch 'master' of https://github.com/gkbroderick/html5-as-a-platform

litteredwitherrors commented on Audio/audio.html:
```diff
@@ -0,0 +1,26 @@
+    <source src="Audio Converts/FillOutSomeForms.ogg" type="audio/mp3">
+    <source src="Audio Converts/FillOutSomeForms.wav" type="audio/mp3">
+    </audio>
+    <br>
```
`br` tags have no semantic meaning, do not use.

litteredwitherrors commented on Audio/audio.html:
```diff
@@ -0,0 +1,26 @@
+    <button onclick="frameChange(-2)" type="button"> Rewind </button>
+    <button onclick="restart()" type="button">Restart</button><br>
+    <button onclick="pauseplay()" type="button">PausePlay</button>
+
```
Remove empty line.

litteredwitherrors commented on Email_Form/email_form.html:
```diff
@@ -0,0 +1,32 @@
+  </head>
+  <body>
+    <main>
+
```
Remove empty line

litteredwitherrors commented on JadeExamples/GalenJade.txt:
```diff
@@ -0,0 +1,102 @@
+!!! 5
```
No `head` or `body`  tags?

litteredwitherrors commented on JadeExamples/WendyJade.txt:
```diff
@@ -0,0 +1,142 @@
+nav.main-nav(role='navigation')
```
Jade file should be save as `.jade`.

litteredwitherrors commented on main.css:
```diff
@@ -0,0 +1,17 @@
+
```
Remove empty line.

gkbroderick committed:
Fix Jade examples add head and body spacing fix text

gkbroderick committed:
Move everthing to central partner folder

gkbroderick committed:
Placing everything in Partner Named Folder

gkbroderick committed:
Added Clearfix class

gkbroderick committed:
Remove <br> tags, add Clearfix, remove empty lines

gkbroderick committed:
remove blank lines

gkbroderick commented:
Thanks for the comments. I unfortunately buried them a little when I made a central "Galen-Wendy" folder for our project. We got together today and cleaned up all the areas you pointed out. We also deleted any unnecessary empty lines of code we found elsewhere. We replaced <br> tags with a Clearfix when necessary.  Lastly, we fixed the improperly converted .jade files (by adding head and body) and their display page.
